src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string;
```

Comment:
i did it the same way as you (only i used 'nothing' for default key value). Its probably mistake and we should use string | null in types and null for Key default value

Reply:
Agree, do it